docs/stencil-docs/storefront-customization/evaluating-color-contrast.md

```diff
@@ -0,0 +1,52 @@
+# Evaluating Color Contrast
+<div class="otp" id="no-index">
+
+### On this page
+- [Performing a manual review](#performing-a-manual-review)
+- [Using an automatic testing tool](#using-an-automatic-testing-tool)
+- [Related resources](#related-resources)
+</div>
+
+Color contrast is the difference between background and foreground content. The colors need to provide maximum contrast so anyone with a vision deficiency can successfully view page information. Color contrast is important for accessibility because it helps people with color blindness, low vision, low contrast vision, and color vision deficiency.
+
+This tutorial will walk you through the steps of evaluating color contrast for your site. Evaluating your site should include performing a manual review and using an automatic testing tool. 
+
+### Prerequisites
+You should be familiar with the [WCAG guidelines](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) on contrast and color requirements.
+
+## Performing a manual review
+Perform a manual review to examine the use of color on your website visually. Ensure the following:
+
+* Text and images are not difficult to read.
+* You do not use color as the only visual means of conveying information.
+* There is no color but black text on white background.
```

Comment:
Can you clarify this sentence? Does this mean that all text should be black on a white background? Or just for certain elements like paragraphs?